internal/pkg/github/webhook.go

```diff
@@ -16,6 +17,10 @@ func StartWatching() {
 // WebhookListener is the endpoint for which the GitHub webhook events should be sent
 // It checks if we support the sent event and handles it accordingly
 func WebhookListener(rw http.ResponseWriter, req *http.Request) {
+	if (strings.Split(req.URL.Path, "/")[2] == "") {
+		rw.WriteHeader(404) // Return 404 Not Found
+		return
+	}
```

Comment:
Maybe using a small library would be easier for future use. We use [github.com/julienschmidt/httprouter](https://github.com/julienschmidt/httprouter) internally for most http routers.